cluster/src/main/java/com/hartwig/pipeline/reruns/StartingPoint.java

```diff
@@ -66,21 +67,12 @@ enum StartingPoints {
                         LinxSomatic.NAMESPACE,
                         VirusAnalysis.NAMESPACE,
                         Chord.NAMESPACE,
+                        LilacBamSlicer.NAMESPACE,
```

Comment:
I think we want to pull this up to an earlier a bit such that we can separate the bam slicing and lilac calling in reruns.